charts/merlin/templates/setup-job.yaml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Any reason for not using Charts Hooks `pre-install` here? https://helm.sh/docs/topics/charts_hooks/

Reply:
We need the vault pod to be up and ready for this job to run. `pre-install` wouldn't work because the vault dependency wouldn't be installed by then, and using `post-install` is also not useful because the helm chart testing tool( internally helm install --wait) doesn’t run the hooks (that are marked for post-install) unless all the pods are ready/healthy.
This is kind of a chicken & egg problem because, in the setup job I am adding the credentials to the vault, which are needed for a successful app start-up.